textual_fspicker/base_dialog.py

```diff
@@ -81,6 +85,14 @@ def __init__(
             select_button: Label for the select button.
         """
         super().__init__()
+
+        if os.name == 'nt':
+            import win32api
+            drivesStr = win32api.GetLogicalDriveStrings()
+            self.drives = drivesStr.split('\x00')[:-1]
+        else:
+            self.drives = "/"
```

Comment:
It seems the intention is that `drives` is a `list[str]`, but here it's been assigned a string.

Also, is there a reason why `drives` would be a "public" value?